zhaquirks/efekta/iaq3.py

```diff
@@ -0,0 +1,363 @@
+        )
+
+
+class RelativeHumidityConfig(CustomCluster, RelativeHumidity):
+    """Relative humidity cluster with calibration offset."""
+
+    cluster_id = RelativeHumidity.cluster_id
+
+    class AttributeDefs(RelativeHumidity.AttributeDefs):
+        """Attribute definitions."""
+
+        humidity_offset: Final = ZCLAttributeDef(id=0x0210, type=t.int16s, access="rw")
+
+
+class CO2ConcentrationConfig(CustomCluster, CarbonDioxideConcentration):
+    """CO2 concentration cluster with configuration attributes."""
+
+    cluster_id = CarbonDioxideConcentration.cluster_id
+
+    class AttributeDefs(CarbonDioxideConcentration.AttributeDefs):
+        """Attribute definitions."""
+
+        # CO2 sensor settings
+        forced_recalibration: Final = ZCLAttributeDef(
+            id=0x0202, type=t.Bool, access="rw"
+        )
+        auto_brightness: Final = ZCLAttributeDef(id=0x0203, type=t.Bool, access="rw")
+        long_chart_period: Final = ZCLAttributeDef(id=0x0204, type=t.Bool, access="rw")
+        set_altitude: Final = ZCLAttributeDef(id=0x0205, type=t.uint16_t, access="rw")
+        factory_reset_co2: Final = ZCLAttributeDef(id=0x0206, type=t.Bool, access="rw")
+        manual_forced_recalibration: Final = ZCLAttributeDef(
+            id=0x0207, type=t.uint16_t, access="rw"
+        )
+
+        # CO2 gas control settings
+        enable_co2: Final = ZCLAttributeDef(id=0x0220, type=t.Bool, access="rw")
+        high_co2: Final = ZCLAttributeDef(id=0x0221, type=t.uint16_t, access="rw")
+        low_co2: Final = ZCLAttributeDef(id=0x0222, type=t.uint16_t, access="rw")
+        invert_logic_co2: Final = ZCLAttributeDef(id=0x0225, type=t.Bool, access="rw")
+
+        # Display settings
+        rotate: Final = ZCLAttributeDef(id=0x0285, type=t.uint16_t, access="rw")
+        internal_or_external: Final = ZCLAttributeDef(
+            id=0x0288, type=t.Bool, access="rw"
+        )
+        night_onoff_backlight: Final = ZCLAttributeDef(
+            id=0x0401, type=t.Bool, access="rw"
+        )
+        automatic_scal: Final = ZCLAttributeDef(id=0x0402, type=t.Bool, access="rw")
+        long_chart_period2: Final = ZCLAttributeDef(id=0x0404, type=t.Bool, access="rw")
+        night_on_backlight: Final = ZCLAttributeDef(
+            id=0x0405, type=t.uint8_t, access="rw"
+        )
+        night_off_backlight: Final = ZCLAttributeDef(
+            id=0x0406, type=t.uint8_t, access="rw"
+        )
+
+
+class AnalogInputCluster(CustomCluster, AnalogInput):
+    """Analog input cluster that relays VOC index to emulated VOC measurement cluster."""
+
+    cluster_id = AnalogInput.cluster_id
+    PRESENT_VALUE = 0x0055
+
+    def _update_attribute(self, attrid, value):
+        """Intercept present_value updates and relay to VOC cluster."""
+        super()._update_attribute(attrid, value)
+        if attrid == self.PRESENT_VALUE and value is not None:
+            self.endpoint.voc_level._update_attribute(MEASURED_VALUE, value)
```

Comment:
Attribute IDs should be pulled from the attribute definition as opposed to relying on a global constant: `EmulatedVOCMeasurement.AttributeDefs.measured_value.id`